torch/testing/_internal/common_quantization.py

```diff
@@ -961,7 +961,7 @@ def forward(self, x):
         out = self.conv2(out)
         out = torch.nn.functional.max_pool2d(out, 2, 2)
         out = self.cat.cat([out, out])
-        out = out.view(-1, 3 * 2 * 2)
+        out = out.reshape(-1, 3 * 2 * 2)
```

Comment:
why is this change needed?

Reply:
Because `view` is broken for channels_last format and there is not easy way to fix that. Although not clear why this PR exposes that particularly.